app/src/main/java/layout/mapFragment.java

```diff
@@ -293,11 +298,15 @@ public void onResume(){
         Log.d("mapTag","on resume fragments");
     }
 */
-    private void testCreateMoodArray() throws ExecutionException, InterruptedException {
-        ElasticSearchController.GetMoodTask getMoodTask = new ElasticSearchController.GetMoodTask();
-        getMoodTask.execute("user");
-        moodArrayList.addAll(getMoodTask.get());
+//ElasticSearchController.GetMoodTask getMoodTask = new ElasticSearchController.GetMoodTask();
+//getMoodTask.execute("user");
+//moodArrayList.addAll(getMoodTask.get());
+
 
+    private void testCreateMoodArray() {
+        //FeelTripApplication.loadFromElasticSearch(); THIS LINE ISN'T NEEDED DUE TO THE WAY WE CALL mapFragment
+        moodArrayList = FeelTripApplication.getMoodArrayList();
+        Log.d("mapTag","test size: "+moodArrayList.size());
```

Comment:
Now properly returns moods from the "Geolocation" filter. Tested and working 100%.